CodeSniffer/Tokenizers/Comment.php

```diff
@@ -54,7 +54,7 @@ public function tokenizeString($string, $eolChar, $stackPtr)
         */
 
         for ($c = 0; $c < $numChars; $c++) {
-            if ($string[$c] !== '/' && $string[$c] !== '*') {
+            if (substr($string, $c, 1) !== '/' && substr($string, $c, 1) !== '*') {
```

Comment:
Old code works perfectly fine on PHP7. The unit tests have been running on it most of the year.